없는숫자더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(numbers) {
+    let answer = 0;
+    let result;
+    for(let i = 0; i < numbers.length; i++) {
+        answer += numbers[i]
+    }
+    result = 45 - answer;
```

Comment:
```suggestion
    let result = 45 - answer;
```
변수선언 을 아래해서 해도 되고
```
return 45 - answer;
```
바로 반환해도 괜찮을 것 같네요